spec/recipes/limits_spec.rb

```diff
@@ -18,20 +18,24 @@
 require_relative '../spec_helper'
```

Comment:
I can make that change. Alternatively, RSpec will [read command line configuration options from `.rspec`](http://www.relishapp.com/rspec/rspec-core/v/3-5/docs/configuration/read-command-line-configuration-options-from-files) so we can put `--require spec_helper` there and remove all of the `require` statements.